src/main/kotlin/org/jitsi/nlj/stats/PacketIOActivity.kt

```diff
@@ -36,6 +36,9 @@ class PacketIOActivity {
     val lastOverallRtpActivity: Instant
         get() = latest(lastRtpPacketReceivedTimestamp, lastRtpPacketSentTimestamp)
 
+    val lastOverallReceivedActivity: Instant
```

Comment:
nit: can you rename this to `lastOverallIncomingActivity`?

Reply:
yes, np